Stripe/STPAddress.m

```diff
@@ -202,6 +202,14 @@ - (BOOL)containsRequiredShippingAddressFields:(PKAddressField)requiredFields {
     return containsFields;
 }
 
+- (BOOL)containsRequiredSEPADebitFields {
+    return (self.line1.length > 0
+            && self.city.length > 0
+            && self.country.length > 0
+            && [STPPostalCodeValidator stringIsValidPostalCode:self.postalCode
```

Comment:
Commented on this on the other PR, but I am suspect that we should be requiring this for IE. We should very what the SEPA api is intended to be, and probably need to have separate postal code requirements for it vs. credit cards if they really require it for all SEPA accounts.